### ISSUE TITLE
[Supporting the platform – Make Code - Sign in with GitHub] 'Use Developer token' link control does not adapt high contrast(Black/White) theme under 'Sign in with GitHub' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3584]
**User Experience**:
Visual impaired user will find difficulty to see the control or element if that control does not adopt the high contrast mode, as they won't be able to take on that controls and they will miss the functionality of such control or element.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to "Your GitHub Repo" option and navigate through the all the controls of dialog "Sign in with GitHub"..
6. Verify that 'Use Developer token' link is adapting high contrast theme or not.

**Actual Result:**
'Use Developer token' link control does not adapt high contrast(Black/White) theme under 'Sign in with GitHub' pop-up.

**Expected Result:**
'Use Developer token' link control should adapt high contrast(Black/White) theme under 'Sign in with GitHub' pop-up. it should look as 'Yellow' color in high contrast black theme and 'Blue' color in high contrast white theme.

**WCAG Reference:**
[WCAG 4.3.1 - No Disruption of Accessibility Features
](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.3.1, #Win10-Edge(Chromium)

![WCAG4 3 1_Use Developer token link](https://user-images.githubusercontent.com/72871089/98107916-31ebf200-1ec1-11eb-8c9d-e339350cd093.jpg)
